Dashboard/app/js/lib/views/reports/export-reports-views.js

```diff
@@ -34,7 +34,18 @@ FLOW.ReportLoader = Ember.Object.create({
 });
 
 FLOW.ExportReportsView = Ember.View.extend({
-  templateName: 'navReports/export-reports'
+  templateName: 'navReports/export-reports',
+  missingSurvey: false,
+  
+  updateSurveyStatus: function(surveyStatus){
+     if (surveyStatus === 'survey-selected') {
+       this.set('missingSurvey',false)
+     }else if (surveyStatus === 'not-selected') {
```

Comment:
the formatting here and on the next else should be with a space i.e. `} else {`

Reply:
okay. let me work on it